src/Components/OATHeader/OATHeader.tsx

```diff
@@ -155,13 +182,31 @@ const OATHeader: React.FC<IOATHeaderProps> = (props) => {
                         state: 'closed'
                     }
                 });
+                // Log import error
+                sendEventTelemetry({
+                    name: TelemetryEvents.importFailException,
+                    triggerType: TelemetryTrigger.UserAction,
+                    appRegion: appRegion,
+                    componentName: componentName,
+                    customProperties: {
+                        errorTitle: error.title,
```

Comment:
Should I do error type then based on the title or something? I'm wondering if we can get some context with this error.

Reply:
I'm not sure what the best way is to get error type. The titles aren't particularly useful and aren't necessarily unique to each problem, but I guess you could try that. It will be different per localization though too which won't give you very good groupings.

Reply:
I'll do unhandled exception, if we need to be more specific, I can dig deeper into how we can achieve that.